### DISCUSSION TITLE
where is the page to add new appstores from

### DISCUSSION THREAD

**Discussion by gjm19761** [IceWhaleTech/CasaOS#1308] [General]
this was mentioned in 0.4.4 changelog three weeks ago

You can add third-party app sources from one page ( the page link will be live ,align with 0.4.4 in this week), offering thousands of applications to choose from.

is this here yet?

**Comment by thedroidkid** [comment]
I think what your looking for is posted here. I added the Linux server app store to mine.

https://github.com/WisdomSky/CasaOS-LinuxServer-AppStore

**Comment by gjm19761** [comment]
thanks but it was mentioned a month ago that there would be a page to install lots of appstores from..

You can add third-party app sources from one page ( the page link will be live ,align with 0.4.4 in this week), offering thousands of applications to choose from.

**Comment by tigerinus** [comment]
https://github.com/IceWhaleTech/CasaOS/assets/7172560/fac1e04a-be74-45b0-9899-24ba886e7053

**Reply by tigerinus** [reply]
checkout awesome.casaos.io for more 3rd party appstores.
